charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,230 @@
+---
+eol-exporter:
+  enabled: true
+  namespaceOverride: ''
+  applicationName: eol-exporter
+  externalSecrets:
+    enabled: false
+  deployment:
+    enabled: true
+    image:
+      repository: saritasallc/eol-exporter
+      tag: prod-b9828d4
+      digest: ''  # if set to a non empty value, digest takes precedence on the tag
+      pullPolicy: IfNotPresent
+    initContainers: []
+    command: ''
+    args: []
+    envFrom: {}
+    env:
+      ENVIRONMENT:
+        value: prod
+      CONFIG_YAML_PATH:
+        value: config.yaml
+      JOB_INTERVAL_HOURS:
+        value: '24'
+    ports:
+      - name: http
+        containerPort: 8080
+        protocol: TCP
+    volumeMounts:
+      config:
+        mountPath: /workspace/app/config.yaml
+        subPath: config.yaml
+    volumes:
+      config:
+        configMap:
+          name: eol-exporter-config
+    startupProbe:
+      enabled: false
+      failureThreshold: 30
+      periodSeconds: 10
+      # Must specify either one of the following field when enabled
+      httpGet: {}
+      exec: {}
+      tcpSocket: {}
+    readinessProbe:
+      enabled: true
+      failureThreshold: 3
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+      initialDelaySeconds: 10
+      # Must specify either one of the following field when enabled
+      httpGet:
+        # Prometheus client WSGI server has a /favicon.ico path that gives a 200 OK response that can
+        # be configured as health check endpoint. See:
+        # https://github.com/prometheus/client_python/blob/master/prometheus_client/exposition.py#L121
+        path: /favicon.ico
```

Comment:
use /metrics URL?

Reply:
Ok, changed:

Context:
Initially, when making API calls for each call to /metrics this would not have been feasible.
Now the /metrics  is much faster so it's possible.

/favicon is still a little faster but negligible:
```

🐳  [PRD] eol-exporter-v1-8599dccbf5-2pkwp app # 
 curl -w "@curl-format.txt" -o /dev/null -s "localhost:8080/metrics"
     time_namelookup:  0.000013s
        time_connect:  0.000068s
     time_appconnect:  0.000000s
    time_pretransfer:  0.000085s
       time_redirect:  0.000000s
  time_starttransfer:  0.001104s
                     ----------
          time_total:  0.001237s
🐳  [PRD] eol-exporter-v1-8599dccbf5-2pkwp app # 
 curl -w "@curl-format.txt" -o /dev/null -s "localhost:8080/favicon.ico"
     time_namelookup:  0.000015s
        time_connect:  0.000079s
     time_appconnect:  0.000000s
    time_pretransfer:  0.000098s
       time_redirect:  0.000000s
  time_starttransfer:  0.000681s
                     ----------
          time_total:  0.000781s

```